recipes/holoplot/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "holoplot" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "2b32b78bea003085413634cb83a8c2a5f48fe41a338056b5b15f5673798c1be4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - bokeh >=0.12.15
+    - holoviews >=1.10.4
+    - pandas
+
+test:
+  imports:
+    - holoplot
+    - holoplot.examples
+    - holoplot.examples.assets
+    - holoplot.examples.user_guide
+    - holoplot.examples.user_guide.images
+    - holoplot.tests
+  requires:
+    - coveralls
+    - flake8
+    - nbsmoke >=0.2.0
+    - nose
+    - parameterized
+    - pytest
```

Comment:
Any thoughts on running the test suite since the dependencies are included?

Reply:
I'm not sure what the conda-forge standard is on this one @jakirkham.  I've seen some recipes run the package unit tests, and I've seen others not.  Do you have any preference one way or another?

Reply:
Normally we put most of our effort into running tests on compiled code. So typically not on Python code. Though this is more a comment on how compiled code is viewed and has little to do with non-compiled code.

It's come up before that we probably shouldn't be listing these test only dependencies if they aren't being exercised. Personally don't find that concerning.

That said, if there are some simple tests we can run, that would probably be ok.

Reply:
@jakirkham, thanks for answering. I personally think that always trying to run at least some quick test suite on a package is valuable, even if those tests are separately run elsewhere during development by the project itself. E.g. tests on a package here on conda-forge could catch that some new dependency has not been added to the recipe on conda-forge when updating to a new source release, or that an updated dependency specification has been missed, or similar. Or the tests could catch that somehow a package does not run properly on "the conda-forge stack" for some reason (e.g. maybe because of tricky low level dependencies, binary incompatibilities between defaults/conda-forge, etc...).

Reply:
So the other challenge that I neglected to mention is managing resources (e.g. build time) vs. package quality. It's hard to say what the right answer is here, but it is something we try to think about. This motivates testing of compiled code in the way that we have.

Reply:
Resources are definitely in my mind when I add tests on conda-forge; that is, I make sure the tests are just a small fraction of the overall build time. 

Not sure what 'small fraction' is actually reasonable, though, given the long build times ;) Previously I added tests that take about 30s out of a 7 minute build (i.e. <10%).